pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ConvertToRawIndexMinionClusterIntegrationTest.java

```diff
@@ -102,6 +102,10 @@ public void testConvertToRawIndexTask()
     File[] indexDirs = tableDataDir.listFiles();
     Assert.assertNotNull(indexDirs);
     for (File indexDir : indexDirs) {
+      // Skip the tmp directory since these are only temporary segments
+      if (indexDir.getName().equals("tmp")) {
```

Comment:
This should be the only thing looking at the `tableDataDir` directly like this. But it does worry me that there was at least 1 assumption that every directory in `tableDataDir` is a segment directory